content/english/python-pixel/C. Activities/b.Activity-2/b.Activity-2.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Modify your color broad"
```

Comment:
Please change the path of the file to content/english/python-pixel/activities/activity-2.md (content/english/python-pixel/C. Activities/b.Activity-2/b.Activity-2.md)